libseccomp/src/version.rs

```diff
@@ -129,10 +131,73 @@ pub(crate) fn ensure_supported_version(msg: &str, expected: ScmpVersion) -> Resu
 #[cfg(test)]
 mod tests {
     use super::*;
+    use std::borrow::Cow;
+
+    struct ScmpVersionTest {
+        pub msg: Cow<'static, str>,
```

Comment:
Is the Co used somewhere? If not it can be a simple `&'static str`.

Reply:
I fixed it. Thanks!